src/Template/Layout/default.ctp

```diff
@@ -15,38 +9,20 @@ echo $this->fetch(
             <?= $this->fetch('title', read('App.title', env('HTTP_HOST'))) ?>
         </title>
 
+        <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta name="author" content="<?= read('App.author') ?>">
+        <?= $this->Html->meta('icon') ?>
         <?= $this->fetch('meta') ?>
-
-
         <?= $this->AssetCompress->css('platform') ?>
-        <?= $this->fetch('css'); ?>
-        <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
-        <!--[if lt IE 9]>
-        <script src="//html5shim.googlecode.com/svn/trunk/html5.js"></script>
-        <script src="//oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>
-        <![endif]-->
+        <?= $this->fetch('css') ?>
+        <?= $this->fetch('headjs') ?>
```

Comment:
The `platform` stylesheet and script are auto-built by AssetCompress.

The rest of the suggestion are nice, bit heavy, but nice. And yes, they look fitting into a separate plugin, not even an RFC (if you ask me). Something that people could use in this application template or the official one or any other.